pom.xml

```diff
@@ -46,7 +46,7 @@
 		<serialization.xstreamVersion>0.2.11</serialization.xstreamVersion>
 		<providermanagementVersion>2.2</providermanagementVersion>
 		<idgenVersion>4.3</idgenVersion>
-		<initializerVersion>1.0.1</initializerVersion>
+		<initializerVersion>1.0.2-SNAPSHOT</initializerVersion>
```

Comment:
Can you also set the Java compliance to Java 8? So that we can use Lambda expressions and get rid of this `intersect` method [here](https://github.com/ivange94/openmrs-module-mksreports/blob/bdbf175b9984a4b1df64c398e6f85576ec877b1e/api/src/main/java/org/openmrs/module/mksreports/dataset/evaluator/ObsSummaryRowDataSetDefinitionEvaluator.java#L49).

Reply:
Ok